### ISSUE TITLE
Regarding SEC. 301- Issues Regarding Ancillary Assets

### ISSUE THREAD

**Issue by chickengiggles** [responsible-financial-innovation-act22/RFIA-bill#6]
Regarding SEC 301. Some folks seem to have issues regarding ancillary assets. 

(I'm assuming ancillary assets are essentially assets such as utility tokens / governance tokens / etc)  

Question is, what needs to be addressed in this section to clarify concerns.

Concerns include the following below:

According to footnotes from this JD Supra article (https://www.jdsupra.com/legalnews/lummis-gillibrand-responsible-financial-7053016/): 

2The RFIA and accompanying materials do not clarify why the drafters have limited “ancillary assets” to those delivered in connection with investment contracts (thereby effectively relying on SEC v W.J. Howey Co. as a predicate to identifying ancillary assets). This approach could potentially limit the utility of the definition for market participants who initially seek to finance DLT platform development only through traditional securities offerings and issue certain token rights in connection with that offering. However, it is notable that the RFIA would not seek to legislatively alter the Howey framework or its application generally, which would have much more wide-ranging implications for financial markets.

Thoughts?

**Comment by Bhaney44** [comment]
I think this is incorrect. Ancillary assets are defined as:

> (A) In general.—The term ‘ancillary asset’ means an intangible, fungible asset that is offered, sold, or otherwise provided to a person in connection with the purchase and sale of a security through an arrangement or scheme that constitutes an investment contract, as that term is used in section 2(a)(1) of the Securities Act of 1933 (15 U.S.C. 77b(a)(1)).

So, Ancillary assets are security tokens, which have additional properties that yield additional regulation. I think the biggest problem with this bill is the adoption of the existing definition of securities from the Securities Act, which is creating so many problems. My own suggestion is to define Digital Securities as, digital assets which represent an equity interest in a company. This definition promotes decentralized innovation and protects investors in line with the public good, _simplex sigillum veri_.

The idea of ancillary assets simply creates unnecessary complexity and does not control governance or utility tokens, which are inherently not securities. I think another great thing this bill could do is provide a clear definition for non-security tokens as assets which have a specific purpose and utility apart from any investment. This would create freedom for innovators and protect them from the arbitrary decision making of executive enforcement agencies that often fail to respect the limits of their jurisdiction, as well as the Constitution. 

By clearly defining non-security tokens as assets not regulated by the SEC, it will ensure that opportunity can remain for open source software projects developing blockchain technologies and digital assets. The reality is that most startups and projects cannot afford the high costs, often six or seven figures, to hire legal counsel to register security assets with the SEC. More generally, most digital assets should not be considered securities because they are not what we typically ascribe to the word security, such as a stock. Instead digital assets are new technologies with a plethora of properties separate and apart from financial investments. It's imperative this reality is respected by the Bill and any new law on the subject.

**Comment by frabaghe** [comment]
Startups controlling a project (often via controlling shares of Proof of Stake) means the project is inherently centralized. It makes no difference if they cannot afford legal counsel as an inability to pay legal fees does not allow them to ignore decades of securities laws. Most of startups issuing cryptocurrencies have limited functioning networks and are marketing themselves as an investment in a common enterprise, with an expectation of profits. Many are just a proof of concept and a white paper. Allowing startups to sell investment products would essentially legalize penny stock scams.

**Comment by Bhaney44** [comment]
> Most cryptocurrencies are Decentralized In Name Only (DINO). Startups controlling a project (often via controlling shares of Proof of Stake) means the project is inherently centralized. It makes no difference if they cannot afford legal counsel as an inability to pay legal fees does not allow them to ignore decades of securities laws. Most of startups issuing cryptocurrencies have limited functioning networks and are marketing themselves as an investment in a common enterprise, with an expectation of profits. Many are just a proof of concept and a white paper. Allowing startups to sell investment products, without the proper disclosures customarily associated with securities, would essentially legalize penny stock scams.

This is completely incorrect. According to the United States Supreme Court, “The test of an investment contract within Securities Act is whether [the] scheme involves an investment of money in a common enterprise with profits to come solely from efforts of others, and, if test is satisfied, it is immaterial whether enterprise is speculative or nonspeculative or whether there is a sale of property with or without intrinsic value.”  In fact, [Securities and Exchange Commission v. W.J. Howey Co et al.](https://supreme.justia.com/cases/federal/us/328/293/), establishes more than seven decades of precedent that to be a security pursuant to the Securities Act, the asset must be designed to produce profits `solely from efforts of others` as part of a `common enterprise`.

Thus, most digital assets are not investment contracts under the Securities Act because most digital assets do not produce any profits `solely from the efforts of others` and often lack a `common enterprise`. Regarding decentralization, this is largely a term of degree and without question most digital assets are far more decentralized than traditional equities. More often than not, startups or open source projects are not selling digital assets as investments, but rather using them as part of an innovative product or service. Therefore, digital assets are more often cryptographic keys than investments and it's important that the bill define the difference. 

> Digital asset securities: only those assets which represent an equity interest in a company or common enterprise and are sold as investments.
> 
> Non-security digital assets: any asset which is decentralized, intended for use within a product or service, and operates as a cryptographic key.
>
